prow/config.yaml

```diff
@@ -24108,6 +24108,47 @@ periodics:
       secret:
         secretName: fejta-bot-token
 
+- interval: 1h
+  agent: kubernetes
+  name: spark-k8s-periodic
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20171218-a18c352b4-1.8
+      imagePullPolicy: Always
+      args:
+      - "--clean"
+      - "--job=$(JOB_NAME)"
+      - "--repo=github.com/apache-spark-on-k8s/spark-integration=cloud-testing-scripts"
+      - "--upload=gs://kubernetes-jenkins/logs/"
+      - "--timeout=75"
```

Comment:
this timeout need to be greater than config.json ( > 180m, probably 200m)

Reply:
Going to do 120m and 150m. That should work right? I don't think it should ever take longer than 2h.

Reply:
you know better how long it would take :-)